src/tools/pub/cmd/pub-deps.md

```diff
@@ -63,18 +63,20 @@ default format.
 * `compact`
 : Prints dependency information as a compact list.
 
+### `--[no-]dev`
+
+Prints all package dependencies, including dev dependencies. 
+Dev dependencies are included by default.
+
```

Comment:
This one isn't too helpful. Please use text like what's in https://parlough-dart-dev.web.app/tools/pub/cmd/pub-downgrade#--no-offline.